cloud-controller-manager/do/firewall_controller.go

```diff
@@ -271,8 +271,9 @@ func (fm *firewallManager) updateFirewall(ctx context.Context, fwID string, fr *
 }
 
 // createReconciledFirewallRequest creates a firewall request that has the correct rules, name and tag
-func (fm *firewallManager) createReconciledFirewallRequest(serviceList []*v1.Service) *godo.FirewallRequest {
+func (fm *firewallManager) createReconciledFirewallRequest(serviceList []*v1.Service) (*godo.FirewallRequest, error) {
 	var nodePortInboundRules []godo.InboundRule
+	healthCheckPorts := make(map[int]struct{})
 	for _, svc := range serviceList {
 		managed, err := isManaged(svc)
```

Comment:
We may need to scope the non-management restriction down to the NodePort logic while continuing to reconcile ports for NLBs.

Withdrawing approval for now so that we don't forget.